doc/source/user/quickstart.rst

```diff
@@ -793,20 +801,22 @@ array along its horizontal axis, either by specifying the number of
 equally shaped arrays to return, or by specifying the columns after
 which the division should occur::
 
-    >>> a = np.floor(10*np.random.random((2,12)))
+    >>> a = np.floor(10*rg.random((2,12)))
     >>> a
-    array([[ 9.,  5.,  6.,  3.,  6.,  8.,  0.,  7.,  9.,  7.,  2.,  7.],
-           [ 1.,  4.,  9.,  2.,  2.,  1.,  0.,  6.,  2.,  2.,  4.,  0.]])
-    >>> np.hsplit(a,3)   # Split a into 3
-    [array([[ 9.,  5.,  6.,  3.],
-           [ 1.,  4.,  9.,  2.]]), array([[ 6.,  8.,  0.,  7.],
-           [ 2.,  1.,  0.,  6.]]), array([[ 9.,  7.,  2.,  7.],
-           [ 2.,  2.,  4.,  0.]])]
-    >>> np.hsplit(a,(3,4))   # Split a after the third and the fourth column
-    [array([[ 9.,  5.,  6.],
-           [ 1.,  4.,  9.]]), array([[ 3.],
-           [ 2.]]), array([[ 6.,  8.,  0.,  7.,  9.,  7.,  2.,  7.],
-           [ 2.,  1.,  0.,  6.,  2.,  2.,  4.,  0.]])]
+    array([[6., 7., 6., 9., 0., 5., 4., 0., 6., 8., 5., 2.],
+           [8., 5., 5., 7., 1., 8., 6., 7., 1., 8., 1., 0.]])
+    # Split a into 3
+    >>> np.hsplit(a,3)
+    [array([[6., 7., 6., 9.],
+           [8., 5., 5., 7.]]), array([[0., 5., 4., 0.],
+           [1., 8., 6., 7.]]), array([[6., 8., 5., 2.],
+           [1., 8., 1., 0.]])]
```

Comment:
Note: we should really consider #11270, which would make this type of output a lot more readable in IPython

Reply:
Yes I agree. This bothered me when I reviewed it, but for now there's no good way to align these outputs better.

Reply:
Not that changing the behavior of `IPython` will change what a regular python REPL (and therefore `refguide`) wants, unfortunately.